content/en/docs/tasks/security/plugin-ca-cert/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Configuring CA Certificates
+description: Shows how operators can configure Istio's CA with an existing root certificate, signing certificate and key.
+weight: 80
+keywords: [security,certificates]
+aliases:
+    - /docs/tasks/security/plugin-ca-cert/
+---
+
+This task shows how operators can configure the Istio certificate authority with an existing root certificate, signing certificate and key.
```

Comment:
```suggestion
This task shows how administrators can configure the Istio certificate authority with an existing root certificate, signing certificate and key.
```

Reply:
"Operator" could mean something different in Istio. Better to replace them with "administrator".

Reply:
Done.